demo/csharp/pose_tracking/Program.cs

```diff
@@ -0,0 +1,199 @@
+                }
+            }
+
+            Cv2.ImShow("pose_tracker", frame);
+            return Cv2.WaitKey(1) != 'q';
+        }
+        static void CvMatToMat(OpenCvSharp.Mat cvMat, out MMDeploy.Mat mat)
+        {
+            mat = new MMDeploy.Mat();
+            unsafe
+            {
+                mat.Data = cvMat.DataPointer;
+                mat.Height = cvMat.Height;
+                mat.Width = cvMat.Width;
+                mat.Channel = cvMat.Dims;
+                mat.Format = PixelFormat.BGR;
+                mat.Type = DataType.Int8;
+                mat.Device = null;
+            }
+        }
+
+        static void PrintHelperMessage()
+        {
+            string message = "usage:\n pose_tracking device det_model pose_model video";
+            Console.WriteLine(message);
+        }
+
+        static void Main(string[] args)
+        {
+            if (args.Length != 4)
+            {
+                PrintHelperMessage();
+                Environment.Exit(1);
+            }
+
+            string device_ = args[0];
+            string det_model_ = args[1];
+            string pose_model_ = args[2];
+            string video = args[3];
+
+            Model det_model = new Model(det_model_);
+            Model pose_model = new Model(pose_model_);
+            Device device = new Device(device_);
+            Context context = new Context(device);
+
+            // initialize tracker
+            PoseTracker tracker = new PoseTracker(det_model, pose_model, context);
+
+            PoseTracker.Params param = new PoseTracker.Params();
+            // set default param
+            param.Init();
+            // set custom param
+            param.DetMinBboxSize  = 100;
+            param.DetInterval = 1;
+            param.PoseMaxNumBboxes = 6;
+            // optionally use OKS for keypoints similarity comparison
+            float[] sigmas = {0.026f, 0.025f, 0.025f, 0.035f, 0.035f, 0.079f, 0.079f, 0.072f, 0.072f,
+                              0.062f, 0.062f, 0.107f, 0.107f, 0.087f, 0.087f, 0.089f, 0.089f };
+            param.SetKeypointSigmas(sigmas);
+
+            // create state
+            PoseTracker.State state = tracker.CreateState(param);
+
+            VideoCapture cap = new VideoCapture(video);
+            if (!cap.IsOpened())
+            {
+                Console.WriteLine("failed to open video: " + video);
+                Environment.Exit(1);
+            }
+
+            int frame_id = 0;
+            OpenCvSharp.Mat frame = new OpenCvSharp.Mat();
+            while (true)
+            {
+                cap.Read(frame);
+                if (frame.Empty())
+                {
+                    break;
+                }
+                CvMatToMat(frame, out var mat);
+                // process
+                PoseTrackerOutput result = tracker.Apply(state, mat);
+
+                // visualize
+                if (!Visualize(frame, result, 1280, frame_id++, true))
```

Comment:
I don't think a hardcode `1280` is a good idea.
If a user-specified size is not an argument for this demo, the original mat size sounds more reasonable